core/src/main/java/org/kohsuke/stapler/export/JSONDataWriter.java

```diff
@@ -28,6 +28,13 @@
 import java.io.Writer;
 import java.lang.reflect.Type;
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Abstract Class after Refactoring switch case
+ */
+
```

Comment:
This comment is false as far as I can tell.  JSONDataWriter is not an abstract class.
```suggestion
```